docs/reference/channel.md

```diff
@@ -508,6 +508,11 @@ that it will cause your pipeline to run forever. Consider using the `take` or `u
 a certain condition is met (e.g. after receiving 10 files, receiving a file named `DONE`).
 :::
 
+:::{note}
+The `channel.watchPath` relies on the Java file watch API which and POSIX notify events.
```

Comment:
```suggestion
The `channel.watchPath` relies on the Java file watch API and POSIX to notify events.
```
I don't think this point makes sense - is this still correct?

Reply:
I will rewrite this note